### DISCUSSION TITLE
Artisan HUB0000_0

### DISCUSSION THREAD

**Discussion by Severnic** [artisan-roaster-scope/artisan#1772] [Q&A]
Hi, I'm a new Artisan user, I have Phidgets that I can control in the Phidget control center. In Artisan, in Config - Events, I set up Sliders to control the motors and burner to test Artisan. I have HUB0000_0 to connect Phidgets to the computer. I am not able to control any of the Phidgets with Artisan. Please advise me on how to connect Artisan to HUB0000_0 and what is the procedure for creating my program in Artisan to control the motors, how to display the temperature, etc. I've already read everything possible, but I can't figure it out. I enjoy working on creating my roaster, but I'm a bit at a dead end now.

**Comment by poundy** [comment]
first thing to know about Phidgets is that you can't run the Phidget software at the same time as any other app that needs to talk to the phidgets. Have you confirmed that?

https://artisan-scope.org/devices/phidgets/ has a lot of information about setting up and using Phidgets. Please elaborate on what you have tried and what you have connected, so others who may be familiar with Phidgets can assist

**Reply by Severnic** [reply]
Hi, thanks for the message, yes I don't have multiple applications running at once, I have phidgets connected to raspberry 4. At first the sliders could control phidgets in Artisan and then I fixed some things and overwrote the settings and it hasn't worked since then. I read the website and use it often. On Hub0000_0 I have 3x Out1001_0 connections to control the burner and two motors (frequency converter), TMP1101_0 to measure ET and BT and REL1100_0 to control another 3 motors (on/off - frequency converter). I don't know how to start over from the beginning to choose everything according to myself, I'm making a mistake somewhere, but I don't know where.

**Reply by Severnic** [reply]
When I connect the USB to windows, I can control the Phidgets with Phidget control center
